consensus/src/dag/dag_handler.rs

```diff
@@ -67,10 +66,35 @@ impl NetworkHandler {
         _buffer: Vec<DAGMessage>,
     ) -> SyncOutcome {
         // TODO: process buffer
+
+        // TODO: Pass bounded executor from EpochManager.
+        let executor = BoundedExecutor::new(2, Handle::current());
+        let epoch_state = self.epoch_state.clone();
+        let mut verified_msg_stream =
+            dag_rpc_rx.concurrent_map(executor, move |rpc_request: IncomingDAGRequest| {
+                let epoch_state = epoch_state.clone();
+                async move {
+                    debug!("deserializing and verifying message");
```

Comment:
these logs doesn't seem very meaningful

Reply:
Added for debugging, will clean up before merging.